docs/networking/diagnostics/diagnosing-network-issues-with-mtr.md

```diff
@@ -5,7 +5,7 @@ author:
 description: 'How to create and interpret MTR reports on your Linux VPS or home computer.'
 keywords: 'mtr,traceroute,latency,loss'
 license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
-alias: ['linux-tools/mtr/']
+alias: ['linux-tools/mtr/', 'networking/diagnosing-network-issues-with-mtr']
```

Comment:
Instead of moving this file, please instead use a symlink so that it can be visible from the category view in both sections.